src/main/resources/openapi/annotations.yaml

```diff
@@ -50,6 +50,33 @@ paths:
             application/json+problem:
               schema:
                 $ref: './problem.yaml'
+  /annotations/{annotation_id}:
```

Comment:
Can you add new endpoints to the postman collection?

Reply:
done